app/controllers/activities_controller.rb

```diff
@@ -1,6 +1,8 @@
 class ActivitiesController < ApplicationController
+  before_action :authenticate_user!
+
  def index
- 	sleep 0.7
+    sleep 0.7
     @activities = PublicActivity::Activity.order("created_at desc").where(recipient_id: current_user).where("owner_id not in (?)", current_user).all
   end
 end
```

Comment:
Make sure that `PublicActivity::Activity` is not nil in your ability model. A simple `puts #{PublicActivity::Activity}` will be enough.
If it is then you will need to include public activity model in ability class. Go through public_acitivity  [doc](https://github.com/chaps-io/public_activity)